CHANGELOG.md

```diff
@@ -241,63 +241,60 @@ installations, reduce memory footprint, and improve performance. We welcome feed
 
 [#2888]: https://github.com/datawire/ambassador/issues/2888
 
-## [1.14.1] (2021-08-24)
-[1.14.1]: https://github.com/emissary-ingress/emissary/releases/v1.14.1
 
-## Emissary-ingress
-
-- Bugfix: Upgraded Envoy to 1.17.4 to address security vulnerabilities CVE-2021-32777, CVE-2021-32778,
-  CVE-2021-32779, and CVE-2021-32781
-
-## [1.14.0] (2021-08-19)
-[1.14.0]: https://github.com/emissary-ingress/emissary/releases/v1.14.0
+## [1.14.1] August 24, 2021
```

Comment:
This switches from `YYYY-MM-DD` for the 2.x versions to `Month DD, YYYY` for the 1.x versions.

Reply:
You're right: I wanted the 1.X versions to be internally consistent, but I actually prefer the ISO8601ish format that we've been using in 2.X.